projects/packages/my-jetpack/_inc/components/product-interstitial-modal/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { Text, Button, ThemeProvider, Col, Container } from '@automattic/jetpack-components';
+import { Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import { useCallback, useState, type FC } from 'react';
+import styles from './style.module.scss';
+
+interface ProductInterstitialModalProps {
+	title: string;
+	hideCloseButton?: boolean;
```

Comment:
It seems like we are not using this prop anywhere, so we can't actually hide the close button in the modal.